src/store/item-store.js

```diff
@@ -51,6 +51,11 @@ store.registerHandler('NEW_ITEM', data => {
 store.registerHandler('NEW_CHECKOUT', data => {
     data.itemAddresses.forEach(address => {
         store.getItemByAddress(address).status = 'CHECKED_OUT';
+        
+        let timestamp = new Date();
+        timestamp.setUTCHours(16);//Set the time to 5pm today;
```

Comment:
Nope. Just figured that was a good tentative time for now. It'll have to be set-able for long term checkouts anyway.